styles.css

```diff
@@ -94,8 +94,7 @@ body {
   .lastRow {
     border: none;
   }
-  
-  @media only screen and (max-width: 768px) {
+  @media only screen and (max-width: 567px) {
```

Comment:
with mobile-first approach we design for the small screens then for larger screens, you should use min-width instead of max-width